packages/delivery-utils/src/integration.ts

```diff
@@ -0,0 +1,7 @@
+import { arrayFromNodeList } from './array-from-node-list';
+
+export function integration(): void {
+	window.evolv.project = {
+		arrayFromNodeList
+	};
```

Comment:
You probably want to do short-circuit assignment here to avoid overriding `window.evolv.project` in the event that it already exists.

Since this project uses TypeScript >4.0, it can use the newer ES2021 compound assignment operators:
```suggestion
window.evolv.project ||= {};
window.evolv.project.arrayFromNodeList ||= arrayFromNodeList;
```

which is equivalent to
```typescript
window.evolv.project = window.evolv.project || {};
window.evolv.project.arrayFromNodeList =  window.evolv.project.arrayFromNodeList || arrayFromNodeList;
```